lib/ignored-paths.js

```diff
@@ -157,6 +173,25 @@ class IgnoredPaths {
                 this.baseDir = path.dirname(path.resolve(options.cwd, ignorePath));
                 addIgnoreFile(this.ig.custom, ignorePath);
                 addIgnoreFile(this.ig.default, ignorePath);
+            } else {
+                try {
+
+                    // if the ignoreFile does not exist, check package.json for eslintIgnore
+                    const packageJSONPath = findPackageJSONFile(options.cwd);
+
+                    if (packageJSONPath) {
+                        const packageJSONOptions = JSON.parse(fs.readFileSync(packageJSONPath, "utf8"));
+
+                        if (packageJSONOptions.eslintIgnore && Array.isArray(packageJSONOptions.eslintIgnore)) {
+                            packageJSONOptions.eslintIgnore.forEach(pattern => {
+                                addPattern(this.ig.custom, pattern);
+                                addPattern(this.ig.default, pattern);
+                            });
+                        }
+                    }
+                } catch (e) {
+                    debug("Could not find package.json to check eslintIgnore property");
```

Comment:
Wondering if we should have multiple try/catch and error logs, in order to distinguish between file not being found vs file was not valid JSON vs whatever other issues.